### ISSUE TITLE
getAll function doesn't retrieve value

### ISSUE THREAD

**Issue by zemoa** [assuncaocharles/ngx-indexed-db#95]
Hello,

I try to use this plugin in my Angular 8 project.
I registered the Module as expected in Readme.
Then I use the service like this :
```
@Injectable({
    providedIn: "roo"'
})
export class RequestCacheHelper {
    constructor(private dbService: NgxIndexedDBService) {
        this.dbService.currentStore = 'request';
    }
...
    retryRequests() {
        this.dbService.getAll().then(
            requestEntityList => {
                if (requestEntityList) {
                    // do something
                }
            });
    }
```
The result "requestEntityList" is always undefined. I look through your code and I discover that in "NgxIndexedDBService" you retrieve the value like that 
```
request.onsuccess = function({ target: { successResult } }: any) {
					resolve(successResult as T[]);
				};
```
whereas it should be, I think :
```
request.onsuccess = function({ target: { success } }: any) {
					resolve(success as T[]);
				};
```

**Comment by assuncaocharles** [comment]
Hi,

Can you create a sandbox reproduction?

**Comment by mauguzun** [comment]
same problem

**Comment by assuncaocharles** [comment]
Hi @mauguzun ,

Can you provide a reproduction repo?

**Comment by mauguzun** [comment]
> Hi @mauguzun ,
> 
> Can you provide a reproduction repo?

Sure

**Comment by assuncaocharles** [comment]
Definitely a bug, I will look into it asap. Meanwhile, PR's are welcome

**Comment by assuncaocharles** [comment]
Hi @mauguzun and @zemoa 

Bug fixed in version 3.0.1

You can check it working here: https://codesandbox.io/s/angular-09u0j

**Comment by mauguzun** [comment]
thank a lot! 
and have a nice day
